### PR TITLE
[RADFISH-105] Update favicons to RADFish logo

### PR DESCRIPTION
**Pull request by melsoriano**
Update the vite config to use the RADFish icons

Homescreen icon screenshot on iPhone:

radfish-icon-screenshot

### CODE REVIEW

**Review by gambinish — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -21,7 +21,7 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <title>RADFish PWA Boilerplate</title>
```
Comment (brockinit) [inline index.html:24]: Can this be changed to just RADFish?

File: index.html [hunk 2]
```diff
@@ -2,11 +2,11 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="icon" type="image/x-icon" href="/favicon.ico" />
+    <link rel="icon" type="image/x-icon" href="/radfish.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="theme-color" content="#000000" />
     <meta name="description" content="Radfish React App" />
-    <link rel="apple-touch-icon" href="/noaafavicon.png" />
+    <link rel="apple-touch-icon" href="/radfish.png" />
```
Comment (brockinit) [inline index.html:9]: Do we need to add a similar icon for android?

**Review by brockinit — CHANGES_REQUESTED** [review]
Left a comment regarding the title, but otherwise looks good!

File: index.html (hunk 2 above)
Comment (melsoriano) [reply index.html:9]: The icons listed in the `vite.config.js` file is used for Android

File: index.html (hunk 1 above)
Comment (melsoriano) [reply index.html]: Changed here: https://github.com/NMFS-RADFish/boilerplate/pull/52/commits/e0c1c2195339765604c804a13d3f8aa9d9948934

**Pull request merged**